src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -415,6 +417,8 @@ public void Initialize()
                     _fileEventsSubscription = EventManager.OfType<FileEvent>()
                         .Where(f => string.Equals(f.Source, EventSources.ScriptFiles, StringComparison.Ordinal))
                         .Subscribe(e => OnFileChanged(e.FileChangeArguments));
+
+                    TraceWriter.Verbose("File event source initialized.");
```

Comment:
I'd remove this - this is just an event subscription

Reply:
Its also creating the event source itself which includes doing file operations. I'm seeing 20-30ms of time spent here. I'll keep it as is.